integrations/gitlab_v2/integration.py

```diff
@@ -0,0 +1,32 @@
+from typing import List, Literal, Union
+
+from port_ocean.core.handlers.port_app_config.api import APIPortAppConfig
+from port_ocean.core.handlers.port_app_config.models import (
+    PortAppConfig,
+    ResourceConfig,
+    Selector,
+)
+from port_ocean.core.integrations.base import BaseIntegration
+from pydantic import Field
+
+
+class GitlabProjectSelector(Selector):
+    onlyGrouped: bool = Field(
+        default=True, description="Retrieve only grouped projects"
+    )
```

Comment:
may I know you meant by grouped projects?

Reply:
The intension was to give users ability to filter only projects that belongs to atleast one group or projects that have existing relationship with group(s).